packages/make-styles/src/runtime/compileStaticCSS.ts

```diff
@@ -1,8 +1,8 @@
 import { MakeStyles } from '../types';
-import { compileCSSRule } from './compileCSS';
+import { compileCSSRules } from './compileCSS';
 import { cssifyObject } from './utils/cssifyObject';
 
 export function compileStaticCSS(property: string, value: MakeStyles): string {
   const cssRule = `${property} {${cssifyObject(value)}}`;
-  return compileCSSRule(cssRule);
+  return compileCSSRules(cssRule)[0];
```

Comment:
What is the story for RTL in static CSS?

Reply:
Automatic transformers are not supported as we don't know how global classes will be applied later. For example:

```css
body { left: 10px }
```